ush/forecast_predet.sh

```diff
@@ -332,19 +332,27 @@ FV3_predet(){
     if [[ "${TYPE}" == "nh" ]]; then  # monotonic and non-hydrostatic
       hord_mt=${hord_mt_nh_mono:-"10"}
       hord_xx=${hord_xx_nh_mono:-"10"}
+      hord_dp=-${hord_xx_nh_nonmono:-"5"}
```

Comment:
Changes you have here would change the behavior of `hord_dp` in NH cases from what is now.
```suggestion
      hord_dp=-${hord_xx_nh_nonmono:-"-10"}
```